### PR TITLE
[cli] Replace unzipper with node-stream-zip

### PR DESCRIPTION
**Pull request by GAntoine**
[node-stream-zip](https://github.com/antelle/node-stream-zip) is more actively maintained and required fewer dependencies. Performance should be exactly the same.

### CODE REVIEW

File: cli/src/lib/cacheRepoUtils.js [hunk 1]
```diff
@@ -105,7 +105,6 @@ export async function ensureCacheRepo(
         }
 
         if (lastUpdated + cacheRepoExpiry < Date.now()) {
-          console.log('• rebasing flow-typed cache...');
```
Comment (Brianzchen) [inline cli/src/lib/cacheRepoUtils.js:108]: Why remove this? Because we have options like skip cache users may want to see this

Comment (GAntoine) [reply cli/src/lib/cacheRepoUtils.js:108]: Good catch, that was removed by accident

**Review by Brianzchen — APPROVED** [review]
LGTM

File: cli/flow-typed/npm/next_v9.x.x.js [hunk 2]
```diff
@@ -0,0 +1,250 @@
+// flow-typed signature: 514646ec2cd38bf48c399f8b3ac0ea8c
+// flow-typed version: 071a03ea49/next_v9.x.x/flow_>=v0.104.x
+
+declare module "next" {
```
Comment (pascalduez) [inline cli/flow-typed/npm/next_v9.x.x.js:4]: Are we using Next.js on the CLI? :)

Comment (GAntoine) [reply cli/flow-typed/npm/next_v9.x.x.js:4]: It must be a dependency of a dependency, as per @Brianzchen recent work.

Comment (Brianzchen) [reply cli/flow-typed/npm/next_v9.x.x.js:4]: I had a look, flowgen which is ships with flow has a dependency on `next`

**Pull request merged**